apps/mosaico/src/Cache/Pubsub.purs

```diff
@@ -0,0 +1,30 @@
+module Mosaico.Cache.Pubsub where
+
+import Prelude
+
+import Control.Promise (Promise, toAffE)
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Data.Maybe (Maybe)
+import Data.Nullable (Nullable, toMaybe)
+import Lettera.Models (CategoryLabel(..))
+import Lettera (LetteraResponse(..))
+
+foreign import data Message :: Type
```

Comment:
Was thinking do we need FFI for reading a `Message`, if we just typed it somehow, but what's the type of `message.data`? Google doesn't tell much about it in their docs either 😅 Only `message.data.toString()`.

Reply:
It's a https://nodejs.org/api/buffer.html which I figured out by adding trace statements to everything.

Reply:
Oh, wonderful. Is it actually then https://pursuit.purescript.org/packages/purescript-node-buffer/8.0.0/docs/Node.Buffer ?

Reply:
I guess, though we'd only use it to call `toString`.